api/tests/opentrons/protocol_engine/clients/test_sync_client.py

```diff
@@ -247,6 +247,40 @@ def test_dispense(
     assert result == response
 
 
+def test_touch_tip(
+    decoy: Decoy,
+    transport: AbstractSyncTransport,
+    subject: SyncClient,
+) -> None:
+    """It should execute a dispense command."""
+    request = commands.TouchTipCreate(
+        params=commands.TouchTipParams(
+            pipetteId="123",
+            labwareId="456",
+            wellName="A2",
+            wellLocation=WellLocation(
+                origin=WellOrigin.BOTTOM,
+                offset=WellOffset(x=0, y=0, z=1),
+            ),
```

Comment:
To save some (keyboard) typing, try to avoid irrelevant data in test objects that are just being used as a value to pass around. Alternatively, construct the the value object such that the actual values are obviously meaningless.

`
```suggestion
            wellLocation=WellLocation(),
```

Or maybe

```python
            wellLocation=WellLocation(
                origin=WellOrigin.BOTTOM,
                offset=WellOffset(x=123, y=456, z=789),
            ),
```
You don't want a reader coming in here at a later date and thinking "oh, the thing that's being tested here must care about the value of the well location", which is a risk if `wellLocation` is set to some "reasonable" value.

The [testdouble wiki](https://github.com/testdouble/contributing-tests/wiki/Meaningless-Test-Data) has some great write ups on the subject.